open_instruct/test_utils.py

```diff
@@ -68,6 +68,14 @@ def test_loading_single_split_with_unit_fractions(self):
         self.assertEqual(len(datasets["test"]), 100)
         self.assertRaises(KeyError, lambda: datasets["train"])
 
+    def test_loading_preference_data(self):
+        dataset_mixer = {
+            "ai2-adapt-dev/ultrafeedback-small": 1000,
+            "ai2-adapt-dev/summarize_from_feedback_small": 1000,
```

Comment:
@natolambert could you move the dataset creationg script in something like `scripts/datasets`

Reply:
@vwxyzjn what exactly do you mean? do you want the get_datasets function in a separate script?